content/zh/docs/concepts/architecture/cloud-controller.md

```diff
@@ -314,46 +170,39 @@ v1/Node:
 - Delete
 
 <!--
-### Route controller
--->
+### Route controller {#authorization-route-controller}
 
-### 路由控制器
-
-<!--
-The route controller listens to Node object creation and configures routes appropriately. It requires get access to Node objects.
+The route controller listens to Node object creation and configures
+routes appropriately. It requires Get access to Node objects.
 -->
+### 路由控制器 {#authorization-route-controller}
 
-路由控制器侦听 Node 对象创建并适当地配置路由，它需要访问 Node 对象。
+路由控制器会监听 Node 对象的创建事件，并据此配置路由设施。
+它需要读取 Node 对象的 Get 权限。
 
-v1/Node:
+`v1/Node`:
 
 - Get
 
 <!--
-### Service controller
--->
-
-### 服务控制器
-
-<!--
-The service controller listens to Service object create, update and delete events and then configures endpoints for those Services appropriately.
--->
+### Service controller {#authorization-service-controller}
 
-服务控制器侦听 Service 对象创建、更新和删除事件，然后适当地为这些服务配置端点。
-
-<!--
-To access Services, it requires list, and watch access. To update Services, it requires patch and update access.
--->
+The service controller listens to Service object Create, Update and Delete events and then configures Endpoints for those Services appropriately.
 
-要访问服务，它需要列表和监视访问权限。要更新服务，它需要修补和更新访问权限。
+To access Services, it requires List, and Watch access. To update Services, it requires Patch and Update access.
 
-<!--
-To set up endpoints for the Services, it requires access to create, list, get, watch, and update.
+To set up Endpoints resources for the Services, it requires access to Create, List, Get, Watch, and Update.
 -->
+### 服务控制器 {#authorization-service-controller}
```

Comment:
```suggestion
### 服务控制器
```
is also OK

Reply:
Added explicit anchor just in case it is referenced from other pages ... we simply don't know, sigh.